lib/editor/actions/active.js

```diff
@@ -413,6 +413,14 @@ export function saveEntity (
     const mappingStrategy = getMapFromGtfsStrategy(component)
     const data = mappingStrategy(entityWithDefaults)
 
+    if (data.location_group_id && data.location_id) {
+      data.location_id = data.location_id.join(',')
+      if (data.is_creating) {
+        delete data.is_creating
+        delete data.id
+      }
+    }
+
```

Comment:
Clicking on a location group in the `EntityList` does not always show the editor for the clicked location group. A console log mentions: "subComponent: 'undefined' is not a valid URL subComponent path" (see line 190), and as a result, the URL is not updated in those instances. The reason for that is that `entityId` is not really defined for a `LocationGroup`, or is it?

Reply:
This causes a strange behavior where the target url is "committed" to the router but not in the browser.

Reply:
I was able to reproduce this once, but then not again. I believe this is a more general datatools bug, as I can't see how location groups differ from any other entity. Do you have any ideas?

Reply:
I've debugged this some more and it appears to be related to the data tools-server version used. Since a lot of the location group server PRs have yet to be merged, I would wait until this PR is officially unblocked before debugging this further.